python/lsst/ts/observatory/control/base_calsys.py

```diff
@@ -116,7 +118,8 @@ async def configure_flat(
         This will include the check the configuration of the telescope, dome,
         and camera in addition to calsys.
         """
-        raise NotImplementedError()
+        await self.configure_projector(filter, wavelength)
```

Comment:
So, this is only calling 2 other methods, there's also no real gain in doing this. Either this is an abstract operation that need to be implemented in the Child classes or it is a concrete operation that does some real work.

Reply:
Ok, just have to figure out how to deal with the camera